### DISCUSSION TITLE
PPM

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [pulsar-edit/.github#99] [Pulsar Support] [answered]
So the documentation says `ppm` is supposed to be installed and, .. it isn't?
Is this just planned and already written down?
I'm trying to update a CI workflow that uses apm to install test packages ..

**Comment by Daeraxa** [comment]
PPM has to be installed as it literally comes with Pulsar.
I suspect that `ppm` and `pulsar` just aren't on the PATH which is a known issue.

**Comment by confused-Techie** [comment, marked as answer]
@ElectronicsArchiver So yes `ppm` must be installed, unless you built from source and didn't update your submodules. What's likely happening like @Daeraxa said is it isn't being put on your PATH properly. We have open issues for this as it's something we need to resolve.

What OS are you one? Since the PATH is only being set properly on Linux at this time. So we can determine if this is part of the known issue

**Reply by ElectronicsArchiver** [reply]
Kubuntu, pulsar is recognized though, just not ppm.

**Reply by ElectronicsArchiver** [reply]
Installed from deb file ( v100 )
